google/api_core/operations_v1/transports/rest_asyncio.py

```diff
@@ -41,17 +39,15 @@
 
 from .base import DEFAULT_CLIENT_INFO as BASE_DEFAULT_CLIENT_INFO, OperationsTransport
 
-
 DEFAULT_CLIENT_INFO = gapic_v1.client_info.ClientInfo(
     gapic_version=BASE_DEFAULT_CLIENT_INFO.gapic_version,
     grpc_version=None,
-    rest_version=auth_version,
+    rest_version=f"rest@{auth_version}",
```

Comment:
```suggestion
    rest_version=f"auth@{auth_version}",
```

Reply:
I've changed the name to `google-auth` which is the publicly available library name.